server/chat-commands/room-settings.ts

```diff
@@ -761,11 +761,15 @@ export const commands: ChatCommands = {
 
 		const titleMsg = Utils.html`Welcome to ${parent ? room.title : user.name}'s` +
 			`${!/^[0-9]+$/.test(title) ? ` ${title}` : ''}${parent ? ' subroom' : ''} groupchat!`;
-		const targetRoom = Rooms.createChatRoom(roomid, `[G] ${title}`, {
+		const defaultSettings = {
 			isPersonal: true,
 			isPrivate: 'hidden',
 			creationTime: Date.now(),
 			modjoin: parent ? null : '+',
```

Comment:
Anyway, do these all seem like things that you _don't_ want to be true for subroom groupchats? Because literally none of them seem like that to me.